nebula2/sclient/GraphStorageClient.py

```diff
@@ -34,8 +34,9 @@ class GraphStorageClient(object):
     DEFAULT_END_TIME = sys.maxsize
     DEFAULT_LIMIT = 1000
 
-    def __init__(self, meta_cache, time_out=60000):
+    def __init__(self, meta_cache, storage_addrs=None, time_out=60000):
         self._meta_cache = meta_cache
+        self.storage_addrs = storage_addrs
```

Comment:
please kindly use `self._storage_addrs = storage_addrs`, as it's intended for internal use only.